### PR TITLE
RM-924 RN 2.18.0

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Preview: https://60ad1f89a3149c0008de2464--pmm-doc-preview.netlify.app/release-notes/2.18.0.html

### CODE REVIEW

File: docs/release-notes/2.18.0.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+
+
+## Bugs Fixed
+
+* [PMM-8132](https://jira.percona.com/browse/PMM-8132): DBaaS: Monitoring of PXC cluster stops working after upgrade from 2.16.0 to 2.18.0
+* [PMM-8127](https://jira.percona.com/browse/PMM-8127): DBaaS: DB Clusters disappear when upgrading from 2.16.0 to 2.18.0
+* [PMM-8058](https://jira.percona.com/browse/PMM-8058): IA: Firing alerts disappear after docker restart
+* [PMM-8069](https://jira.percona.com/browse/PMM-8069): IA: Alert template .yaml is not recognized
+* [PMM-8053](https://jira.percona.com/browse/PMM-8053): 'An unexpected error happened' when accessing Settings page as non-admin
```
Comment (shoffman-percona) [inline docs/release-notes/2.18.0.md:60]:
```suggestion
* [PMM-8053](https://jira.percona.com/browse/PMM-8053): Better error handling for non-admins who try to access the settings page
```

File: docs/release-notes/2.18.0.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+* [PMM-7916](https://jira.percona.com/browse/PMM-7916): DBaaS: Wrong required resources when editing a cluster
+* [PMM-7782](https://jira.percona.com/browse/PMM-7782): Creating new notifications, while older ones are vanishing.
```
Comment (shoffman-percona) [inline docs/release-notes/2.18.0.md:62]: this is resolved as "Won't Fix" and comments in ticket indicate we did nothign for it...should have been ignored by jirabot...

File: docs/release-notes/2.18.0.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+* [PMM-6864](https://jira.percona.com/browse/PMM-6864): PMM2 Oplog Recovery Window dashboard is broken (Thanks to Clyde Shillingford for reporting this issue)
+* [PMM-6543](https://jira.percona.com/browse/PMM-6543): Users logged out of PMM after some period
+* [PMM-7910](https://jira.percona.com/browse/PMM-7910): MongoDB Query Collector never recovers from failed cursor (Thanks to Yann Rouillard for reporting this issue)
+* [PMM-6890](https://jira.percona.com/browse/PMM-6890): The "Environment" filter not working (Thanks to Daniel Guzman Burgos for reporting this issue)
+* [PMM-6451](https://jira.percona.com/browse/PMM-6451): Passing parameters between QAN and dashboards
+* [PMM-5368](https://jira.percona.com/browse/PMM-5368): "Failed to get PMM Server parameter" after config. (Thanks to Martin Wittwer for reporting this issue)
+* [PMM-5135](https://jira.percona.com/browse/PMM-5135): Query Example is often empty (Thanks to Mikhail Solovyev for reporting this issue)
+* [PMM-7673](https://jira.percona.com/browse/PMM-7673): IA: Actions column is transparent
+* [PMM-8123](https://jira.percona.com/browse/PMM-8123): Custom Queries Stop working after Upgrade PMM-Server to 2.18.0
```
Comment (shoffman-percona) [inline docs/release-notes/2.18.0.md:75]: is this a bug or a defect that snuck through and shouldn't be listed?  did this also happen in 2.17.0?

File: docs/release-notes/2.18.0.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+* [PMM-8083](https://jira.percona.com/browse/PMM-8083): PMM-Agent Config is failing
```
Comment (shoffman-percona) [inline docs/release-notes/2.18.0.md:76]: Seems to be an internal issue only...why would the world care?

File: docs/release-notes/2.18.0.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+* [PMM-6553](https://jira.percona.com/browse/PMM-6553): Slow log size units are not defined in help
+* [PMM-6154](https://jira.percona.com/browse/PMM-6154): News panel does not work in fully isolated from internet environment
+* [PMM-5931](https://jira.percona.com/browse/PMM-5931): Graph and values are identical for TOTAL and N/A
+* [PMM-5538](https://jira.percona.com/browse/PMM-5538): Heavy Load with Distinct Queries on Slowlog enabled could cause no data being reported
+* [PMM-8095](https://jira.percona.com/browse/PMM-8095): Community footer link broken
+* [PMM-8088](https://jira.percona.com/browse/PMM-8088): [DBaaS] controller leaves kubeconfig behind in case of error
+* [PMM-7982](https://jira.percona.com/browse/PMM-7982): Query Analytics: Sorting element for the columns is hard to access
+* [PMM-7753](https://jira.percona.com/browse/PMM-7753): Edit DB Cluster shows wrong values by default
+* [PMM-7706](https://jira.percona.com/browse/PMM-7706): LDAP/AD users with Admin role mapping cannot see Integrated Alerting Option from left menu
+* [PMM-7315](https://jira.percona.com/browse/PMM-7315): Ensure DB cluster started to be deleted
```
Comment (shoffman-percona) [inline docs/release-notes/2.18.0.md:87]:
```suggestion
* [PMM-7315](https://jira.percona.com/browse/PMM-7315):  Prevent error message when attempting to list DB clusters that have been requested to be deleted 
```

File: docs/release-notes/2.18.0.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+* [PMM-7184](https://jira.percona.com/browse/PMM-7184): DBaaS: Connection column showing different values after deleting DB cluster
+* [PMM-6676](https://jira.percona.com/browse/PMM-6676): Open Terms and Privacy in New tab
+* [PMM-6552](https://jira.percona.com/browse/PMM-6552): Failure to write config still registers
```
Comment (shoffman-percona) [inline docs/release-notes/2.18.0.md:90]:
```suggestion
* [PMM-6552](https://jira.percona.com/browse/PMM-6552): Do not register with server if config file fails to create on client
```

File: docs/release-notes/2.18.0.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+* [PMM-5220](https://jira.percona.com/browse/PMM-5220): PMM Update for AMI instances shows no Log in Update Popup
+* [PMM-1880](https://jira.percona.com/browse/PMM-1880): Text message while query data is not yet loaded
```
Comment (shoffman-percona) [inline docs/release-notes/2.18.0.md:92]: Looks like a PMM1 issue...not sure why it shows in pmm2.18.0 release notes...

**Review by shoffman-percona — CHANGES_REQUESTED** [review]
Couple language corrections and asked a few questions about things tha tmay need to be hidden.

File: docs/release-notes/2.18.0.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+- Backup Management
+
+    Backup functionality was released as a Technical Preview feature and will require specific prerequisites from the user side to be installed. Currently, PMM will allow you to:
+
+    - manage storage for backups (S3 only);
+    - execute a backup for MySQL and Mongo instances;
+    - restore a MySQL backup to the same instance from where it was taken (via the UI).
+
+    Restore in other cases is not yet implemented on the UI.
```
[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/release-notes/2.18.0.md:39]: was trying to give people very clear expectations .. so they are not looking for features not there yet ..

File: docs/release-notes/2.18.0.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+The goal for this small release was to eliminate some bugs and complete some features.
+
+- DBaaS
+
+    Added the ability for PMM to install the latest versions of the K8s operator into the K8s cluster. There is no longer any need to install the operator manually. Just connect PMM to your K8s cluster and let PMM do the rest.
```
[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/release-notes/2.18.0.md:29]: This is Release notes only,  so yes, changes to Documentation should be in Next branch

File: docs/release-notes/2.18.0.md (hunk 3 above)
Comment (rnovikovP) [reply docs/release-notes/2.18.0.md:75]: this is Defect. added SKIP to it. need to be removed from the RN

File: docs/release-notes/2.18.0.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+* [PMM-6864](https://jira.percona.com/browse/PMM-6864): PMM2 Oplog Recovery Window dashboard is broken (Thanks to Clyde Shillingford for reporting this issue)
+* [PMM-6543](https://jira.percona.com/browse/PMM-6543): Users logged out of PMM after some period
+* [PMM-7910](https://jira.percona.com/browse/PMM-7910): MongoDB Query Collector never recovers from failed cursor (Thanks to Yann Rouillard for reporting this issue)
+* [PMM-6890](https://jira.percona.com/browse/PMM-6890): The "Environment" filter not working (Thanks to Daniel Guzman Burgos for reporting this issue)
+* [PMM-6451](https://jira.percona.com/browse/PMM-6451): Passing parameters between QAN and dashboards
+* [PMM-5368](https://jira.percona.com/browse/PMM-5368): "Failed to get PMM Server parameter" after config. (Thanks to Martin Wittwer for reporting this issue)
+* [PMM-5135](https://jira.percona.com/browse/PMM-5135): Query Example is often empty (Thanks to Mikhail Solovyev for reporting this issue)
+* [PMM-7673](https://jira.percona.com/browse/PMM-7673): IA: Actions column is transparent
+* [PMM-8123](https://jira.percona.com/browse/PMM-8123): Custom Queries Stop working after Upgrade PMM-Server to 2.18.0
```
Comment (rnovikovP) [inline docs/release-notes/2.18.0.md:75]:
```suggestion
```

File: docs/release-notes/2.18.0.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+
+## Bugs Fixed
+
+* [PMM-8132](https://jira.percona.com/browse/PMM-8132): DBaaS: Monitoring of PXC cluster stops working after upgrade from 2.16.0 to 2.18.0
+* [PMM-8127](https://jira.percona.com/browse/PMM-8127): DBaaS: DB Clusters disappear when upgrading from 2.16.0 to 2.18.0
+* [PMM-8058](https://jira.percona.com/browse/PMM-8058): IA: Firing alerts disappear after docker restart
+* [PMM-8069](https://jira.percona.com/browse/PMM-8069): IA: Alert template .yaml is not recognized
+* [PMM-8053](https://jira.percona.com/browse/PMM-8053): 'An unexpected error happened' when accessing Settings page as non-admin
+* [PMM-7916](https://jira.percona.com/browse/PMM-7916): DBaaS: Wrong required resources when editing a cluster
+* [PMM-7782](https://jira.percona.com/browse/PMM-7782): Creating new notifications, while older ones are vanishing.
```
Comment (rnovikovP) [inline docs/release-notes/2.18.0.md:62]:
```suggestion
```

File: docs/release-notes/2.18.0.md (hunk 2 above)
Comment (rnovikovP) [reply docs/release-notes/2.18.0.md:62]: It should not have fixVersion as it's not existed in 2.18 at all. 
removed version from Jira and it's need to be removed from the list too

File: docs/release-notes/2.18.0.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+
+* [PMM-8029](https://jira.percona.com/browse/PMM-8029): DBaaS: PSMDB v1.8 operator support
+* [PMM-7548](https://jira.percona.com/browse/PMM-7548): Integrated Alerting: Disable edit and delete buttons for templates manually staged by user directly on the filesystem
+
+
+
+## Bugs Fixed
+
+* [PMM-8132](https://jira.percona.com/browse/PMM-8132): DBaaS: Monitoring of PXC cluster stops working after upgrade from 2.16.0 to 2.18.0
+* [PMM-8127](https://jira.percona.com/browse/PMM-8127): DBaaS: DB Clusters disappear when upgrading from 2.16.0 to 2.18.0
```
Comment (rnovikovP) [inline docs/release-notes/2.18.0.md:57]:
```suggestion
```
Defect. assed SKIP to it

File: docs/release-notes/2.18.0.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+## Improvements
+
+* [PMM-8029](https://jira.percona.com/browse/PMM-8029): DBaaS: PSMDB v1.8 operator support
+* [PMM-7548](https://jira.percona.com/browse/PMM-7548): Integrated Alerting: Disable edit and delete buttons for templates manually staged by user directly on the filesystem
+
+
+
+## Bugs Fixed
+
+* [PMM-8132](https://jira.percona.com/browse/PMM-8132): DBaaS: Monitoring of PXC cluster stops working after upgrade from 2.16.0 to 2.18.0
```
Comment (rnovikovP) [inline docs/release-notes/2.18.0.md:56]:
```suggestion
```
Deffect. added SKIP and need to be removed from  RN

File: docs/release-notes/2.18.0.md (hunk 4 above)
Comment (rnovikovP) [reply docs/release-notes/2.18.0.md:76]: 🤷‍♂️  looks like an Internal task to me too. Can't understand it by description from user side

Comment (PaulJacobs-percona) [reply docs/release-notes/2.18.0.md:72]: @denisok ?

File: docs/release-notes/2.18.0.md (hunk 7 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.18.0.md:88]: SO... should it be removed? Or is someone looking into it?

**Pull request merged**